index.js

```diff
@@ -296,21 +312,64 @@ function createFocusTrap(element, userOptions) {
   // kind of need to capture the action at the keydown phase.
   function checkTab(e) {
     updateTabbableNodes();
-    if (e.shiftKey && e.target === state.firstTabbableNode) {
-      e.preventDefault();
-      tryFocus(state.lastTabbableNode);
-      return;
+    const isShift = e.shiftKey;
+
+    if (isShift) {
```

Comment:
Could we collapse this into `if (e.shiftKey)`?

Reply:
Since we're moving to the `else` below, this makes even more sense.